db/seeds/hearings.rb

```diff
@@ -6,6 +6,23 @@ module Seeds
   class Hearings < Base
     include PowerOfAttorneyMapper
 
+    # Create the available hearing times and issue counts to pull from
+    AMA_SCHEDULE_TIMES = %w[8:15 9:30 10:15].freeze
```

Comment:
I think you meant to switch these? 
```suggestion
    AMA_SCHEDULE_TIMES = %w[13:30 14:00 15:00 15:15 16:15].freeze
```